.github/workflows/pull-request.yml

```
@@ -0,0 +1,61 @@
+name: Ballerina Observe Internal module build
+
+on: [pull_request]
+
+jobs:
+  ubuntu-build:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+            ./gradlew build
```

Shall we add these flags to all gradle task executions ?
```suggestion
            ./gradlew build  --scan --no-daemon --stacktrace --console=plain
```

---

Fixed with https://github.com/ballerina-platform/module-ballerinai-observe/pull/1/commits/4b2218280291e2dee542697481cd43e7f8469747